dev-docs/source/Testing/MSlice/MSliceTestGuide.rst

```diff
@@ -71,19 +134,124 @@ Taking Cuts
-#. Select ``Generate Script to Clipboard`` and paste the script into the Mantid editor. Please note that on Linux ``Ctrl + V`` might not work as expected. Use ``shift insert`` instead in this case.
-#. Run the script and check that the same slice plot is displayed
+.. figure:: ../../../../docs/source/images/interactive_cuts.png
+   :alt: interactive_cuts.png
+   :align: center
+   :width: 80%
 
-4. Overplot Bragg peaks
+3. Overplot Bragg Peaks
 #######################
 
 #. Navigate to the ``Information`` menu on the cut plot
 #. Select ``Aluminium`` from the submenu for ``Bragg peaks``. Green lines should appear on the cut plot with a respective legend entry.
 #. Deselect ``Aluminium`` form the submenu for ``Bragg peaks``. Both green lines and the respective legend entry should disappear.
 
+.. figure:: ../../../../docs/source/images/cut_with_bragg_peaks.png
+   :alt: cut_with_bragg_peaks.png
+   :align: center
+   :width: 80%
+
+4. Generate a Script
+####################
+
+#. Navigate to the ``File`` menu on the cut plot
+#. Select ``Generate Script to Clipboard`` and paste the script into the Mantid editor. Please note that on Linux ``Ctrl + V`` might not work as expected. Use ``shift insert`` instead in this case.
+#. Run the script and check that the same cut plot is displayed
+
+5. Waterfall Plots
+##################
+
+#. Navigate to the ``Cut`` tab
+#. In the row labelled ``along``, set the ``from`` value to ``0`` and the ``to`` value to ``10``
+#. In the row labelled ``over``, set the ``from`` value to ``-5`` and the ``to`` value to ``5`` as well as the ``width`` value to ``2``
+#. Click ``Plot``. A new window with a cut plot should open.
+#. Click ``Waterfall`` and set the ``x`` value to ``0.5``, then hit enter. The cuts are now plotted with a ``0.5`` offset in direction of the x axis.
+#. Set the ``y`` value to ``2`` and hit enter. The cuts are now plotted with an additional offset (``2``) in direction of the y axis.
+
+.. figure:: ../../../../docs/source/images/waterfall_cut_plot.png
+   :alt: waterfall_cut_plot.png
+   :align: center
+   :width: 80%
+
+The Command Line Interface
+--------------------------
+
+#. Close all plots currently open but not the MSlice interface
+#. Copy the following code into the Mantid editor. You might have to modify the file path for the Load command to the correct location of ``MAR21335_Ei60meV.nxs``.
+
+.. code:: python
+
+    import mslice.cli as mc
+
+    ws = mc.Load('C:\\MAR21335_Ei60meV.nxs')
+    wsq = mc.Cut(ws, '|Q|', 'DeltaE, -1, 1')
+    mc.PlotCut(wsq)
+
+    ws2d = mc.Slice(ws, '|Q|, 0, 10, 0.01', 'DeltaE, -5, 55, 0.5')
+    mc.PlotSlice(ws2d)
+
+#. Run the script.
+#. There should be two new windows with a slice plot and a cut plot
+
+.. figure:: ../../../../docs/source/images/output_mslice_script.png
+   :alt: output_mslice_script.png
+   :align: center
+   :width: 80%
+
+#. Repeat the same test by copying the script into the Jupyter QtConsole of the MSlice interface
```

Comment:
I couldn't find the Jupyter QtConsole so wasn't able to test this. I think maybe some more instructions are needed?